AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/CampaignDetailQueryHandler.cs

```diff
@@ -82,5 +90,13 @@ public CampaignDetailQueryHandler(AllReadyContext context)
 
             return result;
         }
+
+        private CampaignDetailViewModel.CampaignManagerInviteStatus GetCampaignManagerInviteStatus(CampaignManagerInvite campaignManagerInvite)
+        {
+            if (campaignManagerInvite.IsAccepted) return CampaignDetailViewModel.CampaignManagerInviteStatus.Accepted;
+            else if (campaignManagerInvite.IsPending) return CampaignDetailViewModel.CampaignManagerInviteStatus.Pending;
+            else if (campaignManagerInvite.IsRejected) return CampaignDetailViewModel.CampaignManagerInviteStatus.Rejected;
+            else  return CampaignDetailViewModel.CampaignManagerInviteStatus.Revoked;
+        }
```

Comment:
@mgmccarthy Yep, 100% on your guess. And to the second part, I think revoked would simply recall the message. I'd have to double check but once accepted they would be added to the appropriate managers table and therefore the invite is accepted and no longer relevant accept for history. So to remove a manager that would be a separate process updating the managers table. I think! I'm hoping to pull the latest code and refresh my memory a bit tomorrow in readiness for the standup. Been a bit out of it for a fair few weeks :-(